### PR TITLE
fix(invity): cannot select undiscovered account

### LINKED ISSUE #3123: Exchange - does not work for undiscovered accounts?
**Describe the bug**
It seems to me I can't exchange to a coin that has not been previously discovered. The address does not get pre-filled and I can't continue.

**Steps to reproduce:**
1. Go to exchanges
2. Select a coin that you have not discovered yet and has Suite support (such as an empty altcoin account)
3. Create a new account
4. Unable to continue?

**Info:**
 - Suite version: web 20.12.1 (78c9fd3b1ab100fea33c1b63256deae31b64840e)
 - Browser: Mozilla/5.0 (X11; Linux x86_64; rv:83.0) Gecko/20100101 Firefox/83.0
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: model T 2.3.5 regular

**Expected result:**
I can continue with the operation.

**Screenshots:**
c

**Notes:**
I am afraid this is critical enough to cherry-pick into the release.

Cc @trezor/qa. Could you retest on Chrome?

### PR DESCRIPTION
**Pull request by martinboehm**
Will fix #3123

### CODE REVIEW

**Comment by tsusanka** [comment]
We should cherry-pick this into the release, could someone review asap please? Maybe @keraf?

File: packages/suite/src/views/wallet/coinmarket/exchange/offers/Offers/SelectedOffer/components/VerifyAddress/ReceiveOptions/index.tsx [hunk 1]
```diff
@@ -64,20 +64,23 @@ type FormState = {
     extraField?: string;
 };
 
-const VerifyAddressComponent = () => {
+type Props = Pick<UseFormMethods<FormState>, 'setValue'> & {
+    selectedAccountOption: AccountSelectOption | undefined;
```
Comment (keraf) [inline packages/suite/src/views/wallet/coinmarket/exchange/offers/Offers/SelectedOffer/components/VerifyAddress/ReceiveOptions/index.tsx]: Nitpick: `selectedAccountOption: AccountSelectOption | undefined;` could be `selectedAccountOption?: AccountSelectOption;`

**Review by keraf — APPROVED** [review]
Minor nitpick, otherwise looks good!

File: packages/suite/src/views/wallet/coinmarket/exchange/offers/Offers/SelectedOffer/components/VerifyAddress/ReceiveOptions/index.tsx (hunk 1 above)
Comment (martinboehm) [reply packages/suite/src/views/wallet/coinmarket/exchange/offers/Offers/SelectedOffer/components/VerifyAddress/ReceiveOptions/index.tsx]: Done

**Review by tsusanka — APPROVED** [review]

**Comment by tsusanka** [comment]
Let's squash and force push.

**Pull request merged**